python/lsst/ctrl/bps/cli/cmd/commands.py

```diff
@@ -0,0 +1,74 @@
+# This file is part of ctrl_bps.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import click
+from lsst.daf.butler.cli.opt import log_level_option
+from lsst.daf.butler.cli.utils import cli_handle_exception
+from .. import opt
+from .. import script
+
+
+@click.command()
+@opt.config_file_argument(required=True)
+@log_level_option()
+def transform(*args, **kwargs):
+    """Transform a quantum graph to a workflow graph.
+    """
+    pass
```

Comment:
if you're sure you want to leave this in as part of this checkin, maybe you should raise `NotImplementedError` here?